modular_skyrat/modules/synths/code/bodyparts/brain.dm

```diff
@@ -41,11 +41,11 @@
 		last_message_time = world.time
 
 		if(damage > BRAIN_DAMAGE_SEVERE)
-			to_chat(owner, span_warning("Alert: Severe corruption in central processing unit."))
+			to_chat(owner, span_warning("Al停止ユニットで重大な破損が発生しn cessingode: P1_1-H"))
```

Comment:
What?

Reply:
it's a funny - since brain damage is so high their internal reporting is corrupt